fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryInParameterJoinPlan.java

```diff
@@ -105,4 +111,37 @@ public void logPlanStructure(StoreTimer timer) {
         timer.increment(FDBStoreTimer.Counts.PLAN_IN_PARAMETER);
         getInner().logPlanStructure(timer);
     }
+
+    @Nonnull
+    @Override
+    public InternalPlannerGraph rewriteInternalPlannerGraph(@Nonnull List<InternalPlannerGraph> childGraphs) {
```

Comment:
I think that these implementations are quite hard to read without a lot of derails about how the `InternalPlannerGraph`-related classes work. For each of them, perhaps a brief Javadoc can summarize the "intent" (e.g., "Display an IN-join plan in the style of nested loop join...")?